controllers/channels.js

```diff
@@ -7,12 +7,14 @@ import Channel from '../models/channel';
 import Message from '../models/message';
 
 const connection = process.env.SOCKET_SERVER;
+const device = require('express-device');
 
 const decryptUserData = (req) => {
-  JSON.parse(gravity.decrypt(req.session.accessData));
+  return JSON.parse(gravity.decrypt(req.session.accessData));
```

Comment:
This feels like too much is happening in one line. What if there is no req.session? no req.session.accessData? What if the value is not something gravity.decrypt can decrypt? Also, we should probably create a userData DTO to make code more robust.